Lib/socket.py

```diff
@@ -78,7 +78,11 @@
 IntEnum._convert(
         'SocketKind',
         __name__,
-        lambda C: C.isupper() and C.startswith('SOCK_'))
+        lambda C: (C.isupper() and
+                   C.startswith('SOCK_') and
+                   # SOCK_NONBLOCK and SOCK_CLOEXEC are not socket types,
+                   # they are Linux-specific bit flags.
+                   C not in {'SOCK_NONBLOCK', 'SOCK_CLOEXEC'}))
```

Comment:
With this change, we loose the nice repr:

>>> socket.SOCK_CLOEXEC
<SocketKind.SOCK_CLOEXEC: 524288>
>>> socket.SOCK_NONBLOCK
<SocketKind.SOCK_NONBLOCK: 2048>

Which problem are you solving with this change?

Reply:
They are not types of sockets.  They are OS specific special flags. 

If someone prints members of the `SocketKind` to lear what kinds of sockets exist, they will see `SOCK_NONBLOCK` on Linux and will likely be surprised.

Reply:
Maybe put them in a different enum in this case (SocketFlags?), but I would prefer to keep the nice repr, it eases debugging.

Reply:
SocketFlags makes sense. I'll remove this change from this PR, we'll need a new one for it.